plugins/samson_ledger/config/initializers/samson_ledger/client.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
```

Comment:
no need for the client to be in the initializer ... can be required by the plugin and live in lib